examples/fscloud/main.tf

```diff
@@ -63,6 +63,13 @@ module "event_streams" {
   topics                     = var.topics
   existing_kms_instance_guid = var.existing_kms_instance_guid
   metrics                    = ["topic", "partition", "consumers"]
+  quotas = [
+    {
+      "entity"             = "iam-ServiceId-00000000-0000-0000-0000-000000000000",
```

Comment:
Does this value have to be an actual service ID with some kind of permissions on the account? If so, we will need to update the example to create that service ID and grant it the required permissions

Reply:
It does not have to be an actual ID. The Kafka service does not check the ID exists or has permissions.

We use such fake IDs in the terraform-provider-ibm quota data source and resource integration tests, https://github.com/IBM-Cloud/terraform-provider-ibm/blob/7f0333b9b3b51b82e34dfedfa7fa8042eb48587e/ibm/service/eventstreams/data_source_ibm_event_streams_quota_test.go#L16-L25

Reply:
cool - thanks for confirming